networking/metallb/metallb-upgrading-operator.adoc

```diff
@@ -1,31 +1,33 @@
 :_content-type: ASSEMBLY
 [id="metallb-upgrading-operator"]
-= Upgrading the MetalLB Operator
+= Upgrading the MetalLB
 include::_attributes/common-attributes.adoc[]
 :context: metallb-upgrading-operator
 
 toc::[]
 
-The automatic upgrade procedure does not work as expected from {product-title} 4.10 and earlier. A summary of the upgrade procedure is as follows:
+There is a known issue with upgrading from any version of the MetalLB Operator that is version 4.10 or earlier, where automatic updates do not work. However, upgrading from any version of the MetalLB Operator that is version 4.11 or later to a newer version is successful. For example, upgrading from version 4.12 to version 4.13 will occur smoothly.
```

Comment:
I agree with @michaelryanpeter that this line doesn't sound quite right. 
Maybe:
> There is a known issue with upgrading from MetalLB Operator 4.10 or earlier

Reply:
I cannot go with "There is a known issue in versions of the MetalLB Operator released with OCP 4.10 .." strictly speaking the Operator is not released with OCP it is available in the Operator hub OK, but I'm not sure if that means it is released with OCP 4.12. Maybe I am splitting hairs there, but that statement would suggest to me when OCP installed MetalLB is there.